sdk/src/sealance/merkletree-support.ts

```diff
@@ -0,0 +1,237 @@
+import { Field, Plaintext, Poseidon4 } from "../wasm.js";
+import { bech32m } from "@scure/base";
+import { ZERO_ADDRESS } from "../constants.js";
+
+/**
+ * Client library that encapsulates methods for constructing Merkle exclusion proofs for compliant stablecoin programs following the Sealance architecture.
+ *
+
+ * @example
+ * // Construct a Merkle exclusion proof.
+
+ */
+class Sealance {
```

Comment:
All this class does is create a MerkleTree so it should be named as such, can you change it to something like:

```suggestion
class SealanceMerkleTree {
```